tests/test_all_fields.py

```diff
@@ -18,27 +18,20 @@
 #             Fields that are consistently missing during replication
 #             Original Ticket [https://stitchdata.atlassian.net/browse/SRCE-4736]
 KNOWN_MISSING_FIELDS = {
-    'customers':{
-        'tax_ids',
-    },
+    'customers':set(),
     'subscriptions':{
         'payment_settings',
         'default_tax_rates',
         'pending_update',
         'automatic_tax',
     },
     'products':{
-        'skus',
```

Comment:
The response of the latest API(2020-08-27) version does not return `skus` field.